mofacts/client/views/experiment/card.js

```diff
@@ -371,6 +371,17 @@ Template.card.rendered = async function() {
     Session.set('stimDisplayTypeMap', stimDisplayTypeMap);
   }
 
+  // Check if TDF allows for dialogue feedback preferences, if so, route to dialogue feedback widget
+  const allowFeedbackTypeSelect = Session.get('allowFeedbackTypeSelect');
+  const feedbackParamsSet = Session.get('feedbackParamsSet');
+  //Paused Lock to prevent audio from playing during preference selection
```

Comment:
This shouldn't be necessary

Reply:
Agreed. No need for a constant here. I was following the template of some code around it.

Reply:
Oh no, I meant starting on line 377 and the pausedLocks.  Const is preferred if you don't reassign the variable